README.md

```diff
@@ -62,6 +70,18 @@ applications:
   version: v5.1
 ```
 
+### STATUS
+
+| Name | Example Value | Required |
+|------|---------------|----------|
+| LODESTAR_COMPONENT_NAMESPACES | my-namespace1,my-namespace-2 | False |
+| LODESTAR_COMPONENT_NAMES | my-component | False |
+
+Descriptions:
+
+- `LODESTAR_COMPONENT_NAMESPACES` - comma separated list of namespaces containing components to report.  If not specified, no component status will be reported.
+- `LODESTAR_COMPONENT_NAMES` - comma separated list of components to include in the reporting.  If not specified, all components in configured namespaces will be  included in the status report.
+
```

Comment:
Can we add a comment for local dev that the it defaults to http://localhost:8082 since it isnt the conventional port

Reply:
Added section describing the default ports for local dev.